developer-docs/installation/sync_user.md

```diff
@@ -1,14 +1,51 @@
 ---
-title: Syncing User
-page_title: Syncing User
-description: Details on how to sync users
-keywords: sync, user sync, user su=ync job 
+title: Syncing User Data
+page_title: Syncing User Data
+description: Details on how to sync users data
+keywords: sync, user sync, user sync job 
 allowSearch: true
 published: true
 ---
 
 ## Overview
 
+In Sunbird, all write operations are done in Cassandra and read operations are done from Elasticsearch. After write operations are completed in Cassandra, the data is written asynchronously into Elasticsearch as well.
+In case of Cassandra migration that affects all users data (e.g. UserProfileVisibilityReset, UserChannelMigration, etc.), this job is used to sync all user's data from Cassandra to Elasticsearch.
+
 ## Prerequisites
 
-## Syncing User
\ No newline at end of file
+To sync user data, you need:
+
+1. Access to Cassandra database
+2. API Key to access the Sync API
+
+## Configuration Parameters
+
+Pass the following parameters as arguments for the user sync job:
+
+ S.No. | Parameter | Description | Example 
+-------|-----------|-------------|---------
+1 | sunbird_cassandra_server | The IP address of the Cassandra DB. This parameter is used to identify the server on which the Cassandra DB is installed.| 198.168.1.1
+2 | sunbird_cassandra_port | The port number of the Cassandra DB. This parameter is used to identify the port using which the data sync takes place. | 9042 
+3 | sunbird_cassandra_username* | The user name for the Cassandra DB. This parameter is used to authenticate the user accessing the DB. | abc@xyz.com 
+4 | sunbird_cassandra_password* | The password for the Cassandra DB. This parameter is used to authenticate the user accessing the DB.| password 
+5 | sunbird_user_sync_api_endpoint  | The user [Sync API endpoint] (http://docs.sunbird.org/latest/apis/datasyncapi/#tag/Data-Sync-API(s)) | {{domain}}/api/data/v1/index/sync 
+6 | sunbird_user_sync_api_key | The API Key to access the Sync API | As23456789zws34567w234 
+7 | sunbird_user_sync_block_size | The number of user records to be synced per api call | 100 
+8 | sunbird_user_sync_sleep_time | The time interval in milliseconds between api calls | 10000
+
+> Note: If you do not need to authenticate access to the API, pass `""` value for the **username** and **password** parameters.
+
+## Syncing User Data
+
+To sync user data from Cassandra to Elasticsearch:
+
+1. Extract the archive file (sunbird-utils/cassandra-migration-etl/common/UserSyncBin.zip) that contains the script to sync the users data.

```

Comment:
Please provide the exact file path for the archive file. This path does not take to the right file. Which archive is this referring to?

Reply:
@Sunitashelar Updated with Absolute path of the archive file